zproject/settings.py

```diff
@@ -1030,6 +1038,16 @@ def get_secret(key):
             'handlers': ['file'],
             'propagate': False,
         },
+        'django.server': {
+            'propagate': False,
+            'filters': ['skip_200_and_304'],
+        },
+        'django.template': {
+            'handlers': ['console'],
+            'filters': ['require_debug_true'],
+            'level': 'DEBUG',
+            'propgate': False,
+        },
```

Comment:
I think If we set `propagate=True`, the log lines will first be processed by 'django.template' and then 'django' as well. I believe handling of `WARNING` is controlled by `level` argument.